package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "purescript-routing",
+  "version": "0.1.0",
+  "description": "Routing lib for purescript",
+  "main": "index.js",
```

Comment:
I don't think this `main` exists, does it? ;)

Usually I add a `"private": true` to `package.json` for PS projects too, as we don't distribute via npm.